pkg/services/cloudmigration/cloudmigrationimpl/cloudmigration.go

```diff
@@ -597,6 +590,14 @@ func (s *Service) GetSnapshot(ctx context.Context, query cloudmigration.GetSnaps
 		}
 	}
 
+	// For 11.2 we only support core data sources. Apply a warning for any non-core ones before returning to the frontend.
```

Comment:
Moving this here for now, less efficient but this and the `WARNING` type will be going away shortly.